web-ui/public/styles.css

```
@@ -0,0 +1,1129 @@
+  --faint: #616863;
+  --accent: #5bbb91;
+  --accent-hover: #6bc9a0;
+  --accent-soft: rgb(91 187 145 / 12%);
+  --danger: #e77772;
+  --warning: #d9a85e;
+  --complete: #69bd93;
+  --radius: 7px;
+  --header-height: 58px;
+  --font-sans: -apple-system, BlinkMacSystemFont, "Segoe UI", sans-serif;
+  --font-mono: "SFMono-Regular", Consolas, "Liberation Mono", monospace;
+  color: var(--text);
```

Stylelint (17.14.0)

[error] Expected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)